app/templates/empire.hbs

```diff
@@ -9,6 +9,9 @@
     <nav class="nav nav-pills flex-column border-right w-15">
       {{#link-to "empire.capital" class="nav-link"}}
         {{capitalize model.capitalName}}
+        {{#if (and capitalCtl.conquestAvailable capitalCtl.canBuyConquest)}}
```

Comment:
Maybe do the `and` inside `canBuyConquest`? It does not make sense to track if you can buy conquest if conquest isn't available anyway.